opsmngr/agent_api_keys.go

```diff
@@ -0,0 +1,91 @@
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+const (
+	agentAPIKeysBasePath = "groups/%s/agentapikeys"
+)
+
+type AgentAPIKeysService interface {
+	Create(context.Context, string, AgentAPIKeysRequest) (*AgentAPIKey, *atlas.Response, error)
```

Comment:
Forgot to add the body to the create request.

Reply:
can you make this a pointer?